README.md

```diff
@@ -71,17 +71,17 @@ Some examples are provided for direct use in the web browser. If you want to int
 
 You can check the [examples projects](#projects) in this repository or the [Live IDE examples](#live_ide) to know how to bootstrap `bpmn-visualization` in a project.
 
-TypeScript's users should also read the paragraph about the [TypeScript support in the bpmn-visualization README](https://github.com/process-analytics/bpmn-visualization-js/#-typescript-support)
-especially when using bpmn-visualization prior version `0.27.0`. 
+TypeScript's users should also read the paragraph about the [TypeScript support in the `bpmn-visualization` README](https://github.com/process-analytics/bpmn-visualization-js/#-typescript-support)
+especially when using `bpmn-visualization` prior version `0.27.0`. 
 
 
 ## 👁️‍🗨️ Demos
 
 - [Load and Navigation demo](demo/load-and-navigation/index.html) - shows several features of `bpmn-visualization` at the same time. The sources of the demo are available in the
-  [bpmn-visualization](https://github.com/process-analytics/bpmn-visualization-js) repository.
+  [bpmn-visualization-js](https://github.com/process-analytics/bpmn-visualization-js) repository.
```

Comment:
Because it the name of the repository here and not of the library 😉

Reply:
IHMO, we should use the bpmn-visualization name and never use bpmn-visualization-js (or whatever the repository is named on GitHub). This introduces a new name and this is exactly what we want to avoid here. AFAIK, we want a single name when we are talking about the TS lib even if we are talking about its repository or npm package.

If you prefer, we could use "in the repository of [bpmn-visualization].."

Reply:
I'm ok for your proposition 🙂

Reply:
So, I think that putting the link on the word **repository** is more appropriate. What do you think about it?

```suggestion
  [repository](https://github.com/process-analytics/bpmn-visualization-js) of `bpmn-visualization-js` .
```

Reply:
Fine! 😄